LiskHQ/lisk-sdk
# Fix handling of genesis block - Closes #6573

### What was the problem?

This PR resolves #6573 

### How was it solved?

- Update not to hold genesis block in memory
- Update getHighestCommonBlock to getHighestCommonBlockID


### How was it tested?

Run 2 nodes with heavy genesis block (modified mainnet snapshot etc)
- Check 2 nodes will sync even after 500 blocks. (after 500 blocks, genesis block will not be on cache)
- Restart one of the node and check the starting time is significantly faster than 1st time
